server/src/http.rs

```diff
@@ -36,7 +36,7 @@ pub enum Error {
 
     #[snafu(display("Failed to handle request, err:{}", source))]
     HandleRequest {
-        source: crate::handlers::error::Error,
+        source: Box<crate::handlers::error::Error>,
```

Comment:
Do we need to box a concrete error type?

Reply:
it is to fix clippy warning about large fields.

Reply:
👌got it